src/main/java/org/bahmni/insurance/web/RequestProcessor.java

```diff
@@ -152,5 +169,32 @@ public void getOrderCost(HttpServletResponse response) {
 		System.out.println(odooService.getOrderCost("9065024b-9499-4c9b-9a2f-a53f703be2aa"));
 
 	}*/
+	private String validateRequest(String eligibilityRequestValidation) throws IOException {

```

Comment:
@suruchee , please add this validateRequest to more appropirate serviceImpl instead of in controller here.